cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/client/AmazonIdentityManagementRetryClient.java

```diff
@@ -1,23 +0,0 @@
-package com.sequenceiq.cloudbreak.cloud.aws.client;
-
-import com.amazonaws.services.identitymanagement.AmazonIdentityManagement;
-import com.amazonaws.services.identitymanagement.model.SimulatePrincipalPolicyRequest;
-import com.amazonaws.services.identitymanagement.model.SimulatePrincipalPolicyResult;
-import com.sequenceiq.cloudbreak.service.Retry;
-
-public class AmazonIdentityManagementRetryClient extends AmazonRetryClient {
-
-    private final AmazonIdentityManagement client;
-
-    private final Retry retry;
-
-    public AmazonIdentityManagementRetryClient(AmazonIdentityManagement client, Retry retry) {
-        this.client = client;
-        this.retry = retry;
-    }
-
-    public SimulatePrincipalPolicyResult simulatePrincipalPolicy(SimulatePrincipalPolicyRequest request) {
-        return retry.testWith1SecDelayMax5Times(() -> mapThrottlingError(() -> client.simulatePrincipalPolicy(request)));
```

Comment:
you can also remove testWith1SecDelayMax5Times from RetryService then. Probably it also used in a test somewhere (HeartbeatServiceTest or something similar)

Reply:
yea there is probably more cleanup that can be done. There seems to be a lot of pressure to get this fixed so went with minimal changes.